tests/test_auth.py

```diff
@@ -26,81 +26,54 @@ def test_auth_kwargs():
 
 def test_no_credentials_raises():
     with pytest.raises(ValueError):
-        Auth()
+        up42_auth.Auth()
 
 
 def test_should_fail_config_file_not_found(tmp_path):
     config_path = tmp_path / "config_fake.json"
     with pytest.raises(ValueError) as e:
-        Auth(cfg_file=config_path)
+        up42_auth.Auth(cfg_file=config_path)
     assert str(config_path) in str(e.value)
 
 
 def test_should_not_authenticate_with_config_file_if_not_requested():
-    fp = Path(__file__).resolve().parent / "mock_data" / "test_config.json"
-    Auth(cfg_file=fp, authenticate=False)
+    fp = pathlib.Path(__file__).resolve().parent / "mock_data" / "test_config.json"
+    up42_auth.Auth(cfg_file=fp, authenticate=False)
 
 
-def test_should_set_api_host_domain_with_environment(auth_mock):
+def test_should_set_api_host_domain_with_environment(auth_mock: up42_auth.Auth):
     auth_mock.env = "abc"
     assert host.DOMAIN == "abc"
 
 
-def test_get_token(auth_mock):
-    auth_mock._get_token()
-    assert auth_mock.token == TOKEN
+def test_get_workspace(auth_mock: up42_auth.Auth):
+    assert auth_mock.workspace_id == constants.WORKSPACE_ID
 
 
-@pytest.mark.live
```

Comment:
@janchrizz please confirm that it's safe to drop these tests

Reply:
Yes we can drop these auth tests